test/test.googleauth.ts

```diff
@@ -74,8 +119,20 @@ describe('googleauth', () => {
 
   function mockWindows() {
     osStub.returns('win32');
-    mockEnvVar('APPDATA', 'foo');
-    auth._pathJoin = pathJoin;
+    process.env.HOME = '';
```

Comment:
I feel this is bad since it might affects next tests (run by the same process) in some weird way. Maybe `const savedEnv = process.env;` and restore it in `afterAll`?

Reply:
Good eye - but it doesn't!  Checkout out the `beforeEach` hook - the `process.env` you're seeing is actually already entirely mocked out.  This is a fake before it ever gets to this code.

Reply:
Oh, you're right!